tests/requirements.txt

```diff
@@ -1,12 +1,9 @@
-https://github.com/divio/django-cms/archive/release/4.0.x.zip
 coverage
 djangocms_helper
 flake8
 isort
-djangocms_file
-djangocms_picture
-djangocms_audio
-djangocms_video
-https://github.com/divio/djangocms-versioning/archive/master.zip
-https://github.com/FidelityInternational/django-filer/archive/develop.zip
+djangocms-picture==2.3.0
```

Comment:
Can we pin these versions in a less restrictive way, maybe list them as minimum rather than fix?

Reply:
Can't do that, or if we do we need to take into account `ALLOWED_EXTENSIONS` and how that is handled.